OpenEdX/Managers/PushNotificationsManager/Listeners/BrazeListener.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  BrazeListener.swift
+//  OpenEdX
+//
+//  Created by Anton Yarmolenka on 24/01/2024.
+//
+
+import Foundation
+
+class BrazeListener: PushNotificationsListener {
```

Comment:
I guess it should also implement `didReceiveRemoteNotification`

Reply:
https://github.com/openedx/openedx-app-ios/pull/263#discussion_r1482953608